### PR TITLE
cytoband renderings from real data

### PR DESCRIPTION
**Pull request by joeygiordano13**

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -1,20 +1,27 @@
...
+    ],
     "allowJs": true,
     "skipLibCheck": true,
     "esModuleInterop": true,
     "allowSyntheticDefaultImports": true,
     "strict": true,
     "forceConsistentCasingInFileNames": true,
     "noFallthroughCasesInSwitch": true,
-    "module": "commonjs",
+    "module": "esnext",
```
Comment (APherwani) [inline tsconfig.json:16]: Can you change this back to commonjs? @suneettipirneni's script won't run with esnext.

**Review by APherwani — APPROVED** [review]

File: src/components/Chromosome.tsx [hunk 2]
```diff
@@ -18,28 +17,39 @@ function Chromosome({
 }: ChromProps): JSX.Element {
   // Render band within the bandosome
   //console.log(props.selectedLocations);
+  let y = 16;
```
Comment (APherwani) [inline src/components/Chromosome.tsx:20]: Maybe we should have a yaml file or something with these configurations.

**Comment by APherwani** [comment]
Also in chromosome.ts you're importing props but that's not being used. ESLint is causing the build to fail on that account and some other reasons.

File: src/components/Chromosome.tsx (hunk 2 above)
Comment (joeygiordano13) [reply src/components/Chromosome.tsx:20]: we can add this in my next PR

File: tsconfig.json (hunk 1 above)
Comment (joeygiordano13) [reply tsconfig.json:16]: fixed.

**Pull request merged**